apps/calendar/components/calendar/week/WeekColumnsList.tsx

```diff
@@ -135,6 +138,7 @@ const WeekColumnsList = () => {
         if (assigned?.includes(lead.id)) return lead;
       });
     }
+    //Break down the logic into smaller functions or separate utility functions to improve code readability and reusability.
     if (cellContext.appliedBy) {
       leads = leads?.filter((lead) => {
         let joinedPerson = joined.data?.map((join) => {
```

Comment:
//Break down the logic into smaller functions or separate utility functions to improve code readability and reusability.